Packages/com.unity.inputsystem/InputSystem/Plugins/XR/GenericXRDevice.cs

```diff
@@ -23,31 +30,41 @@ protected override void OnRemoved()
         }
     }
 
+    /// <summary>
+    /// The base type for all XR handed controllers.
+    /// </summary>
     [InputControlLayout(commonUsages = new[] { "LeftHand", "RightHand" })]
     public class XRController : InputDevice
     {
+        /// <summary>
+        /// A quick accessor for the currently active left handed device.
+        /// </summary>
+        /// <remarks>If there is no left hand connected, this will be null. This also matches any currently tracked device that contains the 'LeftHand' device usage.</remarks>
```

Comment:
Recommend linking to specific APIs whenever possible. No API docs are more frustrating than Apple's where the guy is clearly writing about some API existing elsewhere but there's no link.